backend/services.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
```

Comment:
I'd move this file up to the root folder for consistency with other docker-compose files